provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/RegisterTask.java

```diff
@@ -101,7 +101,8 @@ public void run(ResponseData responseData, Object context) throws ProvisioningDe
             throw new ProvisioningDeviceClientException(new IllegalArgumentException("authorization cannot be null"));
         }
 
-        //SRS_RegisterTask_25_001: [ Constructor shall save provisioningDeviceClientConfig , securityProvider, provisioningDeviceClientContract and authorization.]
+        // check if using operational CSR
+        this.operationalClientCertificateRequest = provisioningDeviceClientConfig.getOperationalCertificateRequest();
```

Comment:
I don't see the operational certificate being set from the additional data into the provisioning client config. Am I missing something?

It seems like the field should be set in here: https://github.com/Azure/azure-iot-sdk-java/blob/214a54007ede4e23002dc17a3cead02784cd0bf8/provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/ProvisioningDeviceClient.java#L110-L124